tests/output/ConsoleInteractionTest.py

```diff
@@ -49,6 +49,15 @@
 STR_PROJECT_WIDE = "Project wide:"
 
 
+class DummyFileProxy:
+
+    def __init__(self, lines):
+        self.lines = lines
```

Comment:
And put that function in coalib/file/Fileproxy.py itself ? As it seems to be needed in multiple tests